android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -214,17 +214,16 @@ private void hideToolbar() {
     }
 
     public void setNavUpButton() {
-        setNavUpButton(null);
+        setNavUpButton(null, false);
     }
 
     @SuppressWarnings({"ConstantConditions"})
-    public void setNavUpButton(Screen screen) {
+    public void setNavUpButton(Screen screen, boolean isBack) {
```

Comment:
This was a quick update so it can be called from `onConfigurationChanged` since my menu icon was losing its state on rotation.

How should we go about handling this?  Separate functions for styling and setting the state?